osrm.nix

```diff
@@ -11,31 +11,68 @@ in
     packages =
       rec {
         # Patch osrm-backend to use our own car.lua file
-        osrm-backend = pkgs.osrm-backend.overrideAttrs (oa: {
-          # TODO: Use `patches` to directly patch the car.lua file.
-          # Or, use a fork of osrm-backend, and point to it.
-          src = pkgs.runCommandNoCC "osrm-backend-patched-src" { } ''
-            cp -r ${oa.src} $out
-            chmod -R u+w $out
+        patched-osrm-backend = pkgs.stdenv.mkDerivation {
```

Comment:
Why `mkDerivation`? Can't you just do `overrideAttrs` on `pkgs.osrm-backend` pointing `src` to `osrmBackend`?

Reply:
oh sure we can do that let me try that also